README.es.md

```diff
@@ -40,77 +40,79 @@
+ 4. Compilando el código fuente de la app usted mismo. Este es el modo más rápido para obtener nuevas características en su dispositivo, pero es mucho más complicado, así que recomendamos uno de los otros métodos.
+
+Recomendamos el método 1 para la mayoría de usuarios. Los APKs instalados usando método 1 y 2 son compatibles entre sí, pero no lo son con los instalados usando el método 3. Esto es debido a la clave de firmado, ya que los métodos 1 y 2 usan la misma clave (la nuestra), pero el método 3 usa una clave diferente (la de F-Droid). El método 4 excluye totalmente una clave de firmado. Las claves de firmado aseguran que el usuario no esté siendo engañado para instalar/actualizar una APK maliciosa. 
 
-Mientras tanto, si quiere cambiar los fuentes por alguna razón (por ejemplo, la funcionalidad del nucleo de NewPipe se rompe y F-Droid aun no tiene la actualización), recomendamos el siguiente procedimiento:
-1. Repaldear sus datos a través de Ajustes > Contenido > Exporta base de datos para guardar su historia, subscripciones, y listas de reproducción
-2. Desinstalar NewPipe
-3. Descargar el APK del nuevo fuente e instalarlo.
-4. Importar los datos del paso 1 a través de Ajustes > Contenido > Importa base de datos.
+Además, si quiere cambiar el método de instalación por alguna razón (por ejemplo: la funcionalidad del núcleo de NewPipe se rompe o F-Droid aún no publica la actualización), recomendamos el siguiente procedimiento:
+1. Respalde su información a través de Ajustes > Contenido > Exportar base de datos; esto guardará su historial (videos vistos y búsquedas), suscripciones, listas de reproducción y ajustes.
+2. Desinstale NewPipe.
+3. Descargue el APK con un método distinto e instálelo.
+4. Importe la información (la base de datos extraída del paso 1) a través de Ajustes > Contenido > Importar base de datos. Tenga en cuenta que esta opción superpondrá su historial actual (tanto de vídeos como de búsquedas), sus listas de reproducción y (opcionalmente) sus configuraciones.
 
 ## Contribución
-Si tiene ideas, traducciónes, cambios de diseño, limpieza de código, o cambios grandes de código, su ayuda es siempre bienvenida.
-Cuanto más realizamos, mejor se pone la aplicación!
+
+Si tiene ideas, traducciones, cambios de diseño, limpieza de código o cambios grandes de código, su ayuda es siempre bienvenida. ¡Cuanto más hagamos, NewPipe será mucho mejor!
 
 Si quiere involucrarse, fíjese en nuestras [notas de contribución](.github/CONTRIBUTING.md).
 
 <a href="https://hosted.weblate.org/engage/newpipe/es/">
-<img src="https://hosted.weblate.org/widgets/newpipe/es/287x66-grey.png" alt="Estado de la traducción" />
+<img src="https://hosted.weblate.org/widgets/newpipe/es/287x66-grey.png" alt="Translation status" />
```

Comment:
Same thing; why change this back to Engllsh?

Reply:
To keep the same translation of all the alt attributes in the document (all are in English except this one and one other)

I could change all this to Spanish, if necessary.

Reply:
There's no point to `alt` attributes if the reader can't understand them, so yes, please translate them all to Spanish.